slidey_js.js

```diff
@@ -0,0 +1,55 @@
+$(document).ready( function() {
+
+  // If the Giphy API is not working from wherever you're grading this project,
+  // uncomment out the below array and the random number generator further down.
+  // You'll also need to change the variable on line 44 from response.responseJSON...
+  // to images[random].
+  //
+  //
+  // var images = [
+  //   "http://media.giphy.com/media/YLNshs01VYktq/giphy.gif",
+  //   "http://media.giphy.com/media/ClsNWtQIal14c/giphy.gif",
+  //   "http://media.giphy.com/media/47hWogQ3tjW9i/giphy.gif",
+  //   "http://media.giphy.com/media/QPhWqgduVrLRS/giphy.gif",
+  //   "http://media.giphy.com/media/aAuw8HqtaabK/giphy.gif"
+  // ];
+
+
+  // API call
+  // Problem: when I placed this inside of setInterval, the app would read `response`
+  // as undefined.
+  response = $.getJSON( "http://api.giphy.com/v1/gifs/random?api_key=dc6zaTOxFJmzC&rating=pg&tag=bobs+burgers" );
```

Comment:
This is because `$.getJSON` is asynchronous. You have to explicitly use a callback if you want to do anything with the data that's returned.